go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/multiversx/mx-chain-communication-go v1.1.1
 	github.com/multiversx/mx-chain-core-go v1.2.24
 	github.com/multiversx/mx-chain-crypto-go v1.2.12
-	github.com/multiversx/mx-chain-es-indexer-go v1.7.13
+	github.com/multiversx/mx-chain-es-indexer-go v1.7.14-0.20250123094846-b9d3793de7eb
```

Comment:
let's have the proper pre-release after tests

Reply:
added